libp2p/crypto/secp.nim

```diff
@@ -299,8 +300,9 @@ proc getBytes*(key: SkPublicKey): seq[byte] {.inline.} =
 
 proc getBytes*(sig: SkSignature): seq[byte] {.inline.} =
   ## Serialize Secp256k1 `signature` and return it.
-  result = newSeq[byte](SkRawSignatureSize)
-  discard toBytes(sig, result)
+  result = newSeq[byte](72)
+  let length = toBytes(sig, result)
+  result.setLen(length)
```

Comment:
would `length` ever be less than 72 bytes?

Reply:
yep.

Reply:
https://github.com/status-im/secp256k1/blob/be6f5385330905bf1d7cc441be6703cfa7aef847/src/ecdsa_impl.h#L177-L186